subgraphs/aave-v2-forks/protocols/aave-v2/src/mapping.ts

```diff
@@ -79,15 +79,15 @@ import { IPriceOracleGetter } from "../../../generated/LendingPool/IPriceOracleG
 import { Transfer } from "../../../generated/templates/AToken/AToken";
 
 function getProtocolData(): ProtocolData {
-  let letants = getNetworkSpecificConstant();
```

Comment:
lol

Reply:
I think that happened when i was doing a ctrl+f change lmao